src/components/shared/DataTypeSelect.vue

```diff
@@ -42,7 +42,9 @@ export default {
       return filterByAttribute(this.remoteDataTypes?.data || [], this.filters)
     },
     dataTypes() {
-      return mapAttribute(this.filteredDataTypes, 'name')
+      return mapAttribute(this.filteredDataTypes, 'name').filter(
```

Comment:
We are already doing some filtering above so think it should belong in there. We can change filters on line 40 to 

`filters: ({ pipeline }) => (pipeline ? { pipeline, name: 'basecalls and raw data' } : {}),`